cmd/crane/cmd/util.go

```diff
@@ -21,6 +21,35 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 )
 
+type platformsValue struct {
```

Comment:
I could see things like `ko` using this too, is there a good way to share this?

Reply:
We should, but I'd like to play with refactoring `ko` separately before trying to guess the API we want.